pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/SierraTransformer.scala

```diff
@@ -114,8 +114,10 @@ class SierraTransformer(sierraTransformable: SierraTransformable, version: Int)
       edition = SierraEdition(bibData),
       notes = SierraNotes(bibData),
       duration = SierraDuration(bibData),
-      items = SierraItems(itemDataMap)(bibId, bibData) ++
-        SierraElectronicResources(bibId, varFields = bibData.varFields),
+      items =
+        SierraItemsOnOrder(bibId, itemDataMap, orderDataMap) ++
+          SierraItems(itemDataMap)(bibId, bibData) ++
```

Comment:
We should probably be consistent about whether these methods are curried

Reply:
Fixed; I have no idea why SierraItems was being curried.